lib/Eclair/Error.hs

```diff
@@ -251,11 +252,11 @@ style = reAnnotate style
       RuleColor ->
         bold <> color Black
       KindColor isError ->
-        bold <> style (ThisColor isError)
+        bold <> style' (ThisColor isError)
       NoLineColor ->
         bold <> colorDull Magenta
       MarkerStyle st ->
-        bold <> style st
+        bold <> style' st
       CodeStyle ->
         color White
 
```

Comment:
Many changes in this file.. but in the future we will probably swap this out for our own error rendering implementation anyway. :)

Reply:
one of the key changes is the line
```
{-# OPTIONS_GHC -fno-warn-orphans #-}
```
at the top. I am not sure if that is the appropriate fix or if there is another more appropriate alternative.

Reply:
I saw that, but it's fine. Soon we will replace most of this module anyway.